pkg/build/strategies/sti/sti.go

```diff
@@ -338,12 +365,18 @@ func (builder *STI) createCommandForResultingImage(location string) string {
 }
 
 func (builder *STI) createLabelsForResultingImage() map[string]string {
+	generatedLabels := util.GenerateOutputImageLabels(builder.sourceInfo, builder.config)
+	if len(builder.config.RuntimeImage) > 0 {
+		// don't add labels from the builder image if resulting image is based on another image
+		return generatedLabels
+	}
```

Comment:
are we still adding labels from the base runtime image to the resulting runtime image?